src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -186,14 +200,78 @@ export class EventStoreComponent implements OnInit, OnDestroy {
       }
 
       return combinedTimelineData;
+  }  
+
+  private makeIConcurrentEvent(event: DataItem) : IConcurrentEvents {      
```

Comment:
so this might not be necessary. What if instead of making this new object you have IConcurrentEvents inherit(extend) from DataItem, this would let you just pass the dataItems in directly right? You could probably change the reference to IConcurrentEvents to use ITimelineItem  right?